editor/src/clj/editor/pipeline/fontc.clj

```diff
@@ -86,21 +86,48 @@
                            (reductions + 0 (map :glyph-data-size glyph-extents)))]
     glyph-extents))
 
+(defrecord Glyph [character width advance left-bearing ascent descent glyph-cell-wh glyph-data-offset glyph-data-size]
+  IDigestable
+  (digest [_ w]
+    (doto w
+      (.write "#dg/glyph[")
+      (.write (str character))
+      (.write " ")
+      (.write (str width))
+      (.write " ")
+      (.write (str advance))
+      (.write " ")
+      (.write (str left-bearing))
+      (.write " ")
+      (.write (str ascent))
+      (.write " ")
+      (.write (str descent))
+      (.write " ")
+      (.write (str (:width glyph-cell-wh)))
+      (.write " ")
+      (.write (str (:height glyph-cell-wh)))
+      (.write " ")
+      (.write (str glyph-data-offset))
+      (.write " ")
+      (.write (str glyph-data-size))
+      (.write "]"))))
```

Comment:
Font compilation created a lot of throw-away objects when computing the content hash of the font build target because there were so many of these glyph maps created. Using a record here both helps with memory and allows us to provide a more efficient digesting.